src/sb_pointcloud_processing/src/LineExtractorNode.cpp

```diff
@@ -123,7 +123,7 @@ void LineExtractorNode::visualizeClusters() {
     snowbots::RvizUtils::createrMarkerScale(
     this->scale, this->scale, this->scale);
 
-    std::string frame_id = "line_extractor_test";
+    std::string frame_id = "base_link";
```

Comment:
There's a merge conflict here; we should use the changes from #295 (already merged into master), which allows `frame_id` to be configurable from launch file

Reply:
👌